### DISCUSSION TITLE
Performance regression from 9.3.1 to 9.3.3?

### DISCUSSION THREAD

**Discussion by aralroca** [vercel/next.js#11546] [Help] [answered]
I noticed that with the last version of Next.js (9.3.3) the "First load" of all chunks is bigger than 9.3.1

![image](https://user-images.githubusercontent.com/13313058/78128361-0d550680-7416-11ea-8d75-75c7c37d681f.png)

Does anybody have the same problem? Is it something normal or is my problem?

**Comment by timneutkens** [comment, marked as answer]
Nope, no performance regressions. Just a different way of calculating that size, it included `.css` files.

Removed the CSS from the calculation since it led to confusion: https://github.com/zeit/next.js/pull/11401

You can try on next@canary.

**Reply by timneutkens** [reply]
Note that we have extensive bundle size tests and any change of less than or more than 1kB in bundle size will fail the test suite immediately.

On top of that every PR gets a build stats output posted to it that shows the bundle size differences.

This makes it effectively impossible to add unwanted bundle size to the Next.js runtime.

**Reply by aralroca** [reply]
Ok perfect! Thanks for your answer @timneutkens ! Now it's very clear! Thanks!
